apps/web/src/views/Swap/index.tsx

```diff
@@ -115,11 +115,22 @@ export default function Swap() {
   ] = useIsAkkaContractSwapModeActive()
 
   const { chainId } = useActiveWeb3React()
+
+  // Check Independent Field for AKKA
+  useEffect(() => {
+    if (independentField === Field.OUTPUT) {
+      toggleSetAkkaActiveToFalse();
+    }
+    else {
+      toggleSetAkkaActiveToTrue();
+    }
+  }, [independentField])
+  
   // Check if pancakeswap route is better than akka route or not
   useEffect(() => {
     if (akkaRouterTrade?.route?.returnAmountWei && v2Trade?.outputAmount) {
       if (v2Trade?.outputAmount.greaterThan(JSBI.BigInt(akkaRouterTrade?.route?.returnAmountWei))) {
-        toggleSetAkkaModeToFalse()
+        toggleSetAkkaModeToTrue()
```

Comment:
Is this change intentional calling toggleSetAkkaModeToTrue in both cases?

Reply:
just to test AKKA backend ;)